### ISSUE TITLE
Use ossia::net::create_node in the "new node" widget to create nodes in full path

### ISSUE THREAD

**Issue by jcelerier** [ossia/score#417]
e.g. be able to create /foo/bar/baz in one go

**Comment by jcelerier** [comment]
Actually if the device explorer changes, maybe it would be better to rethink how addresses are added ? I'm personnally less and less a fan of pop-up windows but I currently don't have better ideas...

**Comment by bltzr** [comment]
once we have the "attribute sub-inspector", as in @jln- 's mockup, then they could show all default values for the attributes, allow us to edit them, and avoid the need of disgraceful popup windows

**Comment by jcelerier** [comment]
yes, for editing. but for adding I still don't see how to bypass a popup...

**Comment by jln-** [comment]
Maybe something like:

- Hover a node in DE, displays a '+' button
- click '+' button adds a child parameter 
- Newly created parameter is selected and has its name text field selected so one can start typing
- pressing 'enter' creates the parameter for good

If typed name has a slash, pressing 'enter' iterates over this and creates top nodes as needed

Just brainstorming. How does it sound ?

**Comment by bltzr** [comment]
I was thinking more or less the same, but without the + button:
(currently) in order to create a node, a node (or device) must be selected, then we can decide to create a child or a sibling, so basically, we know where it will land.
then, just the same as @jln- proposes:
a new node is created where it should go, and its name is editable
once the name is set, the node is created, with the default attributes (i.e. float type, I'd say), that the user can then edit
